netdiff/parsers/openvpn.py

```diff
@@ -26,20 +27,25 @@ def parse(self, data):
         # add server (central node) to graph
         graph.add_node(server)
         # add clients in graph as nodes
-        for common_name, client in data.client_list.items():
+        for client in data.client_list.values():
             client_properties = {
                 'real_address': str(client.real_address.host),
                 'port': client.real_address.port,
                 'connected_since': client.connected_since.strftime('%Y-%m-%dT%H:%M:%SZ'),
                 'bytes_received': client.bytes_received,
                 'bytes_sent': client.bytes_sent
             }
-            if common_name in data.routing_table:
-                client_properties['local_addresses'] = [
-                    str(data.routing_table.get(common_name).virtual_address)
-                ]
-            graph.add_node(common_name, **client_properties)
+            local_addresses = [
+                str(route.virtual_address)
+                for route in data.routing_table.values()
+                if route.real_address == client.real_address
+                if not isinstance(route.virtual_address, (
+                    ipaddress.IPv4Network, ipaddress.IPv6Network))
```

Comment:
Can we remove this check @nemesisdesign? Checking for the instances? This has been suggested by @tonyseek